PowerEditor/src/WinControls/StaticDialog/StaticDialog.cpp

```diff
@@ -24,7 +24,7 @@ StaticDialog::~StaticDialog()
 	if (isCreated())
 	{
 		// Prevent run_dlgProc from doing anything, since its virtual
-		::SetWindowLongPtr(_hSelf, GWLP_USERDATA, NULL);
+		::SetWindowLongPtr(_hSelf, GWLP_USERDATA, 0);

```

Comment:
Semantic problem: it could be `nullptr` but not `0`.

Reply:
`SetWindowLongPtr` function's 3rd argument is `LONG_PTR` which is not a pointer but an integer:
https://docs.microsoft.com/en-us/windows/win32/api/winuser/nf-winuser-setwindowlongptra